bundles/binding/org.openhab.binding.denon/src/main/java/org/openhab/binding/denon/internal/DenonGenericBindingProvider.java

```diff
@@ -53,10 +53,10 @@ public String getBindingType() {
      */
     @Override
     public void validateItemType(Item item, String bindingConfig) throws BindingConfigParseException {
-        if (!(item instanceof SwitchItem || item instanceof DimmerItem || item instanceof StringItem)) {
+        if (!(item instanceof SwitchItem || item instanceof StringItem)) {
             throw new BindingConfigParseException("item '" + item.getName() + "' is of type '"
                     + item.getClass().getSimpleName()
-                    + "', only Switch-, Dimmer- and StringItems are allowed - please check your *.items configuration");
```

Comment:
Restore to original. Also apply to the other similar throws.

Reply:
This and other cases have been restored to original